### PR TITLE
CR#27107 Updated dependencies

### PR DESCRIPTION
**Pull request by DreierF**
Addresses issue [TS-27107](https://cqse.atlassian.net/browse/TS-27107)

- [x] Changes are tested adequately
- [-] Agent's README.md updated in case of user-visible changes
- [x] CHANGELOG.md updated
- [-] Present new features in [N&N](https://wiki.cqse.eu/pages/viewpage.action?pageId=689566)
- [-] [TGA Tutorial](https://docs.teamscale.com/tutorial/setting-up-tga-java) updated
- [-] [TIA Tutorial](https://docs.teamscale.com/tutorial/tia-java/) updated

Please respect the vote of the [Teamscale bot](https://demo.teamscale.com) or flag irrelevant findings as tolerated or false positives. If you feel that the Teamscale config needs adjustment, please state so in a comment and discuss this with your reviewer.

### CODE REVIEW

File: teamscale-gradle-plugin/build.gradle [hunk 1]
```diff
@@ -25,9 +25,8 @@ dependencies {
 	implementation project(':teamscale-client')
 	implementation project(':report-generator')
 	implementation gradleApi()
-	implementation 'org.jetbrains.kotlin:kotlin-stdlib:1.3.41'
-	implementation 'org.eclipse.jgit:org.eclipse.jgit:5.0.2.201807311906-r'
-	implementation 'com.squareup.retrofit2:converter-moshi:2.5.0'
+	implementation 'org.eclipse.jgit:org.eclipse.jgit:5.11.1.202105131744-r'
```
Comment (stahlbauer) [inline teamscale-gradle-plugin/build.gradle:28]: Seems to be fine. Solves CVE-2020-13956 (HttpClient).

**Review by stahlbauer — APPROVED** [review]
Thanks for upgrading the affected dependencies. For me, everything looks fine. Having separate commits/pull requests for changes that are not related to the issue would be perfect. However, we might not gain much of it in this case.

File: .idea/codeStyles/Project.xml [hunk 2]
```diff
@@ -26,7 +26,15 @@
...
+      <option name="CALL_PARAMETERS_WRAP" value="5" />
+      <option name="CALL_PARAMETERS_LPAREN_ON_NEXT_LINE" value="true" />
+      <option name="CALL_PARAMETERS_RPAREN_ON_NEXT_LINE" value="true" />
+      <option name="METHOD_PARAMETERS_WRAP" value="5" />
+      <option name="METHOD_PARAMETERS_LPAREN_ON_NEXT_LINE" value="true" />
+      <option name="METHOD_PARAMETERS_RPAREN_ON_NEXT_LINE" value="true" />
+      <option name="EXTENDS_LIST_WRAP" value="1" />
+      <option name="METHOD_CALL_CHAIN_WRAP" value="1" />
+      <option name="ASSIGNMENT_WRAP" value="1" />
     </codeStyleSettings>
```
[the remark this answers is not in the document]
Comment (DreierF) [reply .idea/codeStyles/Project.xml:38]: Yeah that is correct. The new IntelliJ just keeps inserting these settings and it did not seem worth to me creating a separate ticket for that.

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,7 @@ We use [semantic versioning](http://semver.org/):
 - PATCH version when you make backwards compatible bug fixes.
 
 # Next Release
+- [feature] Support for Java 16 (and experimental support for Java 17)
```
[the remark this answers is not in the document]
Comment (DreierF) [reply CHANGELOG.md:8]: We added the support by updating JaCoCo that we internally use to the latest version

File: teamscale-gradle-plugin/build.gradle (hunk 1 above)
Comment (DreierF) [reply teamscale-gradle-plugin/build.gradle:28]: Thanks for re-checking!

**Pull request merged**